blackboxopt/optimizers/testing.py

```diff
@@ -148,6 +150,81 @@ def is_deterministic_with_fixed_seed(
     return True
 
 
+def is_deterministic_when_reporting_shuffled_evaluations(
+    optimizer_class: Union[
+        Type[SingleObjectiveOptimizer], Type[MultiObjectiveOptimizer]
+    ],
+    optimizer_kwargs: dict,
+) -> bool:
+    """Check if determinism isn't affected by the order of initially reported data.
+
+    Repeatedly initialize the optimizer with the same parameter space and a fixed seed.
+    Report a set of initial evaluations in randomized order as initial data. Start
+    optimizing and check if the generated configurations for all optimizers are equal.
+
+    Args:
+        optimizer_class: Optimizer to test.
+        optimizer_kwargs: Expected to contain additional arguments for initializing
+            the optimizer. (`search_space` and `objective(s)` are set automatically
+            by the test.)
+
+    Returns:
+        `True` if the test is passed.
+    """
+
+    space = ps.ParameterSpace()
+    space.add(ps.ContinuousParameter("p1", [0, 1]))
```

Comment:
The bounds needs to be a tuple, not a list `(0, 1)` (I know we're not extremely good at sticking to this, but it's [true](https://github.com/boschresearch/parameterspace/blob/main/parameterspace/parameters/continuous.py#L25)).

Reply:
Done